src/compile/data/filterinvalid.ts

```diff
@@ -20,27 +20,40 @@ export class FilterInvalidNode extends DataFlowNode {
     super(parent);
   }
 
-  public static make(parent: DataFlowNode, model: UnitModel): FilterInvalidNode {
-    const {config, mark, markDef} = model;
+  public static make(
+    parent: DataFlowNode,
+    model: UnitModel,
+    dataSourcesForHandlingInvalidValues: DataSourcesForHandlingInvalidValues
+  ): FilterInvalidNode {
+    const {config, markDef} = model;
 
-    const invalid = getMarkPropOrConfig('invalid', markDef, config);
-    if (invalid !== 'filter') {
+    const {marks, scales} = dataSourcesForHandlingInvalidValues;
+    if (marks === 'pre-filter' && scales === 'pre-filter') {
```

Comment:
I'm not sure if `pre-filter` is a used term in VL, but I found `pre-filter` hard to understand without context, especially when we are not using enum here and it's hard for us to see the comment (if any) explaining the meaning. 

The `pre` part is not clear to me. `pre` which filter? 
IIUC, `pre-filter` means pre-null-filter. It's easier to understand for me to call it `include-nulls`.

Reply:
renamed.